### PR TITLE
add changelog entries for s3 and twingate bugs

### PR DESCRIPTION
**Pull request by ollevche**
Related to #1380 and #1357.

This PR adds missing changelog entries for these issues:

* [Crash when using a conditional with Twingate resource](https://github.com/opentofu/opentofu/issues/1380)

* [Backend/S3: Regression in workspace-prefix - must have permission to default env: workspace prefix](https://github.com/opentofu/opentofu/issues/1357)

## Target Release

1.7.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ BUG FIXES:
 * Don't show false update action when import resource with sensitive datasource([#1220](https://github.com/opentofu/opentofu/pull/1220))
 * Fix panic when provisioner source and content are both null ([#1376](https://github.com/opentofu/opentofu/pull/1376))
 * Fix large number will be truncated in plan ([#1382](https://github.com/opentofu/opentofu/pull/1382))
+* Fix S3 backend must have permissions to default 'env:' workspace prefix ([#1445](https://github.com/opentofu/opentofu/pull/1445))
```
Comment (Yantrio) [inline CHANGELOG.md:52]: I'd be tempted to say that the changelog should talk about the positive effect here.

```suggestion
* S3 backend no longer requires to have permissions to use the default 'env:' workspace prefix ([#1445](https://github.com/opentofu/opentofu/pull/1445))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -49,6 +49,8 @@ BUG FIXES:
...
+* Fix crash when using a conditional with Twingate resource ([1446](https://github.com/opentofu/opentofu/pull/1446))
```
Comment (Yantrio) [inline CHANGELOG.md:53]:
```suggestion
* Fixed a crash when using a conditional with Twingate resource ([1446](https://github.com/opentofu/opentofu/pull/1446))
```

Comment (ollevche) [reply CHANGELOG.md]: Thanks! Here it is - 182df0bba9c91852721b41298da29ac44d2e9213

File: CHANGELOG.md (hunk 1 above)
Comment (ollevche) [reply CHANGELOG.md]: Thanks! Here it is - 182df0bba9c91852721b41298da29ac44d2e9213

**Review by Yantrio — APPROVED** [review]

**Pull request merged**